### DISCUSSION TITLE
flask render_template("index.html")

### DISCUSSION THREAD

**Discussion by melo1993** [pallets/flask#4846] [Q&A]
Hi, boss
I have a problem, render_template("index.html"), the index.html have other html files like 
<img width="657" alt="image" src="https://user-images.githubusercontent.com/26834632/195801391-d5a278de-70e3-496e-b602-7ff3d3180309.png">

I cann't open the /dirname/index.html , can you help me check it, best thanks
<img width="562" alt="image" src="https://user-images.githubusercontent.com/26834632/195800866-7d02be99-39bb-4eae-85a3-3169ca0a8d8a.png">
<img width="1033" alt="image" src="https://user-images.githubusercontent.com/26834632/195803699-f2fec0da-e28e-4057-9dd6-7a8682e43720.png">

**Comment by tachyondecay** [comment]
You seem to be confused about **templates** versus **routes/endpoints.**

You have defined a single route, `/`, that maps to the `index()` endpoint in your app. So when you visit `localhost:5000/` in your browser, the app calls `index()` to handle that. 

In your case, you've told your `index` endpoint to render the `index.html` template.

If you want to follow a link from `/` to `/dirname`, you need to define a route in your app that maps `/dirname` to another endpoint.

You should probably review the [Quickstart](https://flask.palletsprojects.com/en/2.2.x/quickstart/).
